bpf_image.c

```diff
@@ -86,10 +93,14 @@ static const char *offsets[SKF_AD_MAX] = {
 #ifdef SKF_AD_ALU_XOR_X
 	[SKF_AD_ALU_XOR_X] = "xor_x",
 #endif
-#ifdef SKF_AD_VLAN_TAG
+#ifdef NPCAP_AD_VLAN_TAG
+	[NPCAP_AD_VLAN_TAG] = "vlan_tci",
+#elif defined(SKF_AD_VLAN_TAG)
```

Comment:
If at most one of {`NPCAP_AD_VLAN_TAG`, `SKF_AD_VLAN_TAG`} is defined in any one build, using `#elif defined(NPCAP_AD_VLAN_TAG)` would keep the diff cleaner.

Reply:
As mentioned above, both may be defined currently for a Npcap build. That is why the `NPCAP_AD_*` check must come first. However, we have not made a SDK release for that yet, so it is still open to change.

Reply:
Thank you, let me check again.